api/v1alpha1/envoygateway_types.go

```diff
@@ -512,6 +512,14 @@ type ExtensionManager struct {
 	//
 	// +optional
 	FailOpen bool `json:"failOpen,omitempty"`
+
+	// GRPCMaxMessageSizeBytes defines the maximum message size in bytes that can be
```

Comment:
I assume that we didn't support http, so `MaxMessageSizeBytes` or `MaxMessageSize` is fine?